miso-web/src/main/webapp/scripts/box_visualization.js

```diff
@@ -696,12 +696,14 @@
     hyperlinkifyBoxable: function(name, id, text) {
       var path = "/miso/";
       var prefix = name.substring(0, 3);
-      if (prefix == "SAM" || prefix == "LIB" || prefix.substring(1, 3) == "PO") {
+      if (prefix == "SAM" || prefix == "LIB" || prefix == "LDI" || prefix.substring(1, 3) == "PO") {
         if (prefix == "SAM") {
           path += "sample/";
         } else if (prefix == "LIB") {
           path += "library/";
-        } else {
+        } else if (prefix == "LDI") {
+          path += "libraryaliquot/";
+        } else if (prefix == "IPO") {
```

Comment:
Surrounding `if` condition should be changed to match this (`IPO` instead of `?PO`)... Or refactor to get rid of the surrounding `if` statement

Reply:
Done.